vktrace/vktrace_layer/vktrace_lib_trim.cpp

```diff
@@ -1066,7 +1066,7 @@ void snapshot_state_tracker() {
                         copyRegion.bufferRowLength = 0;    //< tightly packed texels
                         copyRegion.bufferImageHeight = 0;  //< tightly packed texels
                         copyRegion.bufferOffset = lay.offset;
-                        copyRegion.imageExtent.depth = 1;
+                        copyRegion.imageExtent.depth = imageIter->second.ObjectInfo.Image.extent.depth >> i;
```

Comment:
Does this need to be 'max(1, imageIter->second.ObjectInfo.Image.extent.depth >> i)' to avoid setting copyRegion.imageExtent.depth to 0?  Same for width and height below.

Reply:
It doesn't need, these data are also depth, width and height of that image, trim get the data through capturing VkImageCreateInfo when target app call vkCreateImage, they are already verified at the image creation by the target running environment.

Reply:
This may be true for 3D images, but for 2D and 1D images, where depth is 1, the right shift will produce a depth of 0 for mip levels greater than 0.  This would also be the case for 1D images where height is equal to 1.

Reply:
Yes, you are right. Depth must be 1 for 2D image, depth and height must be 1 for 1D image ( https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/VkBufferImageCopy.html ). I'll submit another commit for it.